src/amo/components/MoreInfo.scss

```diff
@@ -0,0 +1,38 @@
+@import "~core/css/inc/mixins";
+@import "~core/css/inc/vars";
+
+.MoreInfo-header {
+  @include addonSection();
+
+  border-top-left-radius: $border-radius-default;
+  border-top-right-radius: $border-radius-default;
+  font-size: $font-size-default;
+  margin-bottom: 1px;
+  text-align: center;
+}
+
+.MoreInfo-contents {
+  @include addonSection();
+
+  border-bottom-left-radius: $border-radius-default;
+  border-bottom-right-radius: $border-radius-default;
+  font-size: $font-size-s;
+
+  dt {
+    font-weight: bold;
+    line-height: 1.2;
```

Comment:
Actually, not a SASS trick! https://developer.mozilla.org/en/docs/Web/CSS/line-height#Prefer_unitless_numbers_for_line-height_values